internal/nostr/nostr.go

```diff
@@ -728,28 +727,13 @@ func (svc *Service) startSubscription(ctx context.Context, subscription *Subscri
 			svc.stopSubscription(subscription)
 			return
 		}
-		time.Sleep(time.Duration(waitToReconnectSeconds) * time.Second)
 		relay, isCustomRelay, err = svc.getRelayConnection(ctx, subscription.RelayUrl)
 		if err != nil {
-			// TODO: notify user about relay failure
-			waitToReconnectSeconds = max(waitToReconnectSeconds, 1)
-			waitToReconnectSeconds = min(waitToReconnectSeconds * 2, 900)
-			svc.Logger.WithError(err).WithFields(logrus.Fields{
-				"subscription_id": subscription.Uuid,
-				"relay_url":       subscription.RelayUrl,
-			}).Errorf("Failed to connect to relay, retrying in %vs...", waitToReconnectSeconds)
 			continue
 		}
 
 		sub, err := relay.Subscribe(ctx, []nostr.Filter{*filter})
 		if err != nil {
-			// TODO: notify user about subscription failure
```

Comment:
I think we should still log an error here, shouldn't we?

Reply:
This error never happened (in absolute terms), that's why I was confident of it, but doesn't hurt to log, should Ii add it back?